kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-api/kie-wb-common-stunner-client-api/pom.xml

```diff
@@ -104,6 +104,11 @@
       <artifactId>appformer-client-api</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.kie.kogito</groupId>
+      <artifactId>lienzo-core</artifactId>
```

Comment:
mmm I am not sure why we need to add the lienzo dependency into the stunner's client api module. The idea is that lienzo is the library used for the implementation of our client side stuff, but our "client API" should be decoupled from lienzo. This way, at some point, we should be able to move to another technology for rendering our client side stuff.

I don't think this blocks this PR, but it would be nice analyzing why adding the dependency to lienzo in our API module, and trying to see if it's easy to revert this and keep the API decoupled from the implementation library (lienzo). This can be done in a separate PR, but please, try to do this analysis and let's see why the need of this depenendency here. This creates lot of coupling and I would prefer to fix it at some point.

Thanks!